docker-compose-examples/grist-with-postgres-redis-minio.yml

```diff
@@ -0,0 +1,88 @@
+# This examples shows how to start up Grist that:
+# - Uses Postgres as a home database,
+# - Redis as a state store.
+# - MinIO for snapshot storage
+
+# It is STRONGLY RECOMMENDED not to use this container in a way that makes it accessible to the internet.
+# This setup lacks basic security or authentication.
```

Comment:
Auth is something people really want, probably ranking over external db / redis / snapshots.

This is certainly a useful example though!

Reply:
Added an Auth example with OIDC. :)